pos-print/src/test/java/com/clouway/pos/print/adapter/http/ReportServiceTest.kt

```diff
@@ -43,96 +43,103 @@ class ReportServiceTest {
       }
     })
 
-    val reply = reportService.printOperatorReport(request.mockRequest(operatorReportDTO))
+    val reply = reportService.printOperatorReport(request.mockRequest(getOperatorReport()))
```

Comment:
request.mockRequest smells. 


The SitebricksRequestMockery could become: FakeRequest and companion object to be added:

```kotlin
  companion object Factory {
    fun <E> newRequest(value: E) {
      
    }
  }
```

so the usage will become: FakeRequest.newRequest or just newRequest()